azure-spring-boot-samples/azure-active-directory-spring-boot-sample/README.md

```diff
@@ -6,6 +6,7 @@ The sample is composed of two layers: Angular JS client and Spring Boot RESTful
 
 #### Application.properties
 You need to have an registered app in your Azure AD tenant and create a security key.
+If you are using [Azure China](https://docs.microsoft.com/en-us/azure/china/china-welcome), configure property `azure.activedirectory.instance=cn`, the default value without configuration is `global`.
```

Comment:
for example, if your azure account is in format xxx@xxx.partner.onmschina.cn

Reply:
Readme updated.